MapboxNavigation/PollyVoiceController.swift

```diff
@@ -96,7 +95,8 @@ public class PollyVoiceController: RouteVoiceController {
             input.voiceId = voiceId
         }
         
-        input.text = "<speak><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>\(text)</prosody></speak>"
+        // input.text = "<speak><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>\()</prosody></speak>"
```

Comment:
This will render the volume and speaking rate options nonfunctional. What should we do about those options until API support for them lands?